### PR TITLE
Cdn management cli

### PR DESCRIPTION
**Pull request by csmengwan**
The content to be added to Changelog is as follows:
* 
* 
*

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@csmengwan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (qinwa). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by amarzavery** [comment]
file lib/commands/arm/cdnmanagement/cdnmanagement._js
- Line 91 '~~Get and~~ Show info**r**mation  **about **a specific cdn profile'
- Line 160  option('--subscription is mentioned twice
- In general -s is a shorthand reserved switch for --subscription
- while creating an endpoint, -n should should be used for --endpoint-name. You can use -e in other places where you are using the endpoint (Look at group create -h and group deployment create -h for better understanding)
- description for profile delete, endpoint purge and load commands is wrong.

.option('-d, --is-compression-enabled [is-compression-enabled]', $('Is the compression enabled for the Cdn')) 
726     .option('-u, --is-http-allowed [is-http-allowed]', $('Is the http traffic allowed for the Cdn')) 
727     .option('-s, --is-https-allowed [is-https-allowed]', $('Is the https traffic allowed for the Cdn')) 
What are the values that you accept? Yes/No 0/1 True/False. Please document one set so users know what to pass.

**Comment by amarzavery** [comment]
Line 446 'List endpoints by profile **and resource group**'

**Comment by amarzavery** [comment]
origin commands should use -n for name instead of -o and the long form switch should be --name and not --origin-name. Please apply this rule for all your commands to give a consistent experience with other commands.

File: lib/commands/arm/cdnmanagement/cdnmanagement.utils.js [hunk 1]
```diff
@@ -33,7 +33,7 @@ exports.getOriginNamesString = function (originList) {
 
 //Convert command input boolean string to boolean
 exports.getBooleanFromString = function (booleanString) {
-	if (booleanString == 'true') {
+	if (booleanString.toLowerCase() == 'true') {
```
Comment (amarzavery) [inline lib/commands/arm/cdnmanagement/cdnmanagement.utils.js]: In javascript we use === (http://stackoverflow.com/questions/359494/does-it-matter-which-equals-operator-vs-i-use-in-javascript-comparisons)

File: lib/commands/arm/cdnmanagement/cdnmanagement._js [hunk 2]
```diff
@@ -1264,13 +1263,13 @@ exports.init = function(cli) {
   origin.command('set <origin-name> <endpoint-name> <profile-name> <resource-group>')
     .description($('Update origin of the given origin name, endpoint name, profile name, and resource group'))
     .usage('[options] <origin-name> <endpoint-name> <profile-name> <resource-group> [host-name] [http-port] [https-port]')
-    .option('-o, --origin-name <origin-name>', $('Name of the origin'))
+    .option('-n, --origin-name <origin-name>', $('Name of the origin'))
```
Comment (amarzavery) [inline lib/commands/arm/cdnmanagement/cdnmanagement._js]: --name

File: lib/commands/arm/cdnmanagement/cdnmanagement._js [hunk 3]
```diff
@@ -1426,7 +1425,7 @@ exports.init = function(cli) {
   customDomain.command('show <custom-domain-name> <endpoint-name> <profile-name> <resource-group>')
     .description($('Get custom domains by custom domain name, endpoint name, profile name, and resource group'))
     .usage('[options] <endpoint-name> <profile-name> <resource-group>')
-    .option('-c, --custom-domain-name <custom-domain-name>', $('Name of the custom domain'))
+    .option('-n, --custom-domain-name <custom-domain-name>', $('Name of the custom domain'))
```
Comment (amarzavery) [inline lib/commands/arm/cdnmanagement/cdnmanagement._js]: --name. Consistency. Make sure you do these changes for all the commands

**Comment by amarzavery** [comment]
LGTM

**Comment by amarzavery** [comment]
Are there any more updates or should I merge the PR ?

**Comment by csmengwan** [comment]
No more updates on this, I will try to get some time to work on the post/patch long time run fix

**Comment by amarzavery** [comment]
@csmengwan - REsolve conflicts

**Pull request merged**